### DISCUSSION TITLE
powered down and up, display no longer working

### DISCUSSION THREAD

**Discussion by kcossabo** [Blackymas/NSPanel_HA_Blueprint#729] [Q&A]
I got the panel 100% (or I thought) working. 
- sensors were showing
- multip pages
- lights switches on other pages
- QR Code

I was rocking and rolling. 

I powered it down, and now I am in this state;
- powers up, showing HA screen.
- TFT 3.2.2 
- ESPHome 3.2.2
- Physical buttons still control the automation mapped to
- I can from the device page reboot it
- screen stays on 'loading...' 
- ESPHome page shows device online

I go to the 'scene' and go to the blueprint for the device (one being used)
- change a 'Sensor' to a different sensor
- or even date format

Screen updates and back working
[logs_nspanelone_logs.txt](https://github.com/Blackymas/NSPanel_HA_Blueprint/files/11332778/logs_nspanelone_logs.txt)

Fully reproducable. 

a Bug, or user error?

**Comment by edwardtfn** [comment]
Looks like something in your system isn't responding in a reasonable time during the construction of the Home page, so it would be nice if you can take a look at your Home Assistant logs and try to find some clue.
The blueprint should handle those situations, so I would say it's also a bug. If you manage identifying some entity causing this issue, please share here as that could help making the blueprint more robust.
Lots of those items have already been fixed in the version currently in `dev`, so I believe the next release will be more robust to this anyways.

**Reply by edwardtfn** [reply]
Please take a look at this to understand better the reasons for this "Loading..." screen: https://github.com/Blackymas/NSPanel_HA_Blueprint/discussions/671

**Reply by kcossabo** [reply]
which log would be of use?

Home Assistant Core shows 

```
Logger: aioesphomeapi.connection
Source: runner.py:179 
First occurred: April 22, 2023 at 3:52:23 PM (14 occurrences) 
Last logged: 12:19:37 PM

nspanelone @ 10.10.101.195: Connection error occurred: [Errno 104] Connection reset by peer
nspanelone @ 10.10.101.195: Connection error occurred: Ping response not received after 90.0 seconds
nspanelone @ 10.10.101.195: Connection error occurred: Connection lost
```

when I power cycle it.

**Reply by BluThunder2k** [reply]
Try changing anything in the blueprint and press save.   Mine will hang on that screen and any change to the blueprint gets it working after a power cycle.
